arviz/plots/backends/matplotlib/pairplot.py

```diff
@@ -75,6 +126,18 @@ def plot_pair(
                 **divergences_kwargs
             )
 
+        if point_estimate:
+            pe_x = calculate_point_estimate(point_estimate, x)
+            pe_y = calculate_point_estimate(point_estimate, y)
+            if diagonal:
+                ax_hist_x.axvline(pe_x, **point_estimate_kwargs)
+                ax_hist_y.axhline(pe_y, **point_estimate_kwargs)
+
+            ax.axvline(pe_x, **point_estimate_kwargs)
+            ax.axhline(pe_y, **point_estimate_kwargs)
+
+            ax.scatter(pe_x, pe_y, marker="s", s=figsize[0] + 50, **point_estimate_kwargs, zorder=4)
```

Comment:
I think scatter and axvline have some kwarg incompatibilities, I am not sure they should share kwargs. Maybe there could be a warning in docs (along the lines of explaining where are kwargs eventually passed to)